python/tests/test_reduce.py

```diff
@@ -0,0 +1,89 @@
+import pytest
+import torch
+from torch.testing import assert_close
+
+import triton
+import triton.language as tl
+
+
+def patch_kernel(template, to_replace):
+    kernel = triton.JITFunction(template.fn)
+    for key, value in to_replace.items():
+        kernel.src = kernel.src.replace(key, value)
+    return kernel
+
+
+@triton.jit
+def reduce1d_kernel(x_ptr, z_ptr, block: tl.constexpr):
+    x = tl.load(x_ptr + tl.arange(0, block))
+    tl.store(z_ptr, tl.OP(x, axis=0))
+
+
+@triton.jit
+def reduce2d_kernel(x_ptr, z_ptr, block_m: tl.constexpr, block_n: tl.constexpr):
+    range_m = tl.arange(0, block_m)
+    range_n = tl.arange(0, block_n)
+    x = tl.load(x_ptr + range_m[:, None] * block_n + range_n[None, :])
+    z = tl.OP(x, axis=AXIS)
```

Comment:
Here `AXIS` should be `axis` which is defined in the `reduce2d_config`?

Since flake8 report the F821 error for the usage of `AXIS` here.

If it is due to the stupidness of flake8, we may use this
https://flake8.pycqa.org/en/3.1.1/user/ignoring-errors.html
to explicitly ignore the flake8 errors.

Reply:
Done